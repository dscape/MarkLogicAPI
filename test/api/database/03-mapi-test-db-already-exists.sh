#!/bin/sh
curl -svX PUT localhost:6984/mapitest -d "" 2>&1 \
  | sed -e 's/^\*.*//' \
  | grep '[<|{].[A-C|E-Z|a-z]'