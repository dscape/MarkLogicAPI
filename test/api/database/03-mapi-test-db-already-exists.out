{"error":"file_exists","reason":"The database could not be created, the file already exists."}